cc/dual_net/dual_net_test.cc

```diff
@@ -272,5 +271,34 @@ TYPED_TEST(DualNetTest, TestBackendsEqual) {
   }
 }
 
+TEST(WouldCaptureTest, WouldCapture) {
```

Comment:
can you test this same position with w to play?

Reply:
done